src/Cache/Cache.php

```diff
@@ -383,12 +362,11 @@ public static function readMany(array $keys, string $config = 'default'): array
      */
     public static function increment(string $key, int $offset = 1, string $config = 'default')
     {
-        $engine = static::pool($config);
         if (!is_int($offset) || $offset < 0) {
```

Comment:
The `is_int()` check can now be removed since the argument is typehinted. Also for `$offset < 0` it would be better to throw an `InvalidArgumentException` now.